### ISSUE TITLE
Blinking cursor by default

### ISSUE THREAD

**Issue by sutram** [akermu/emacs-libvterm#98]
Great package. I have been using it regularly.

I think after https://github.com/akermu/emacs-libvterm/pull/88 was merged, I noticed that the cursor in my emacs session has started blinking.

Is it possible to change that behavior back to not blink by default?

Right now, as a workaround, I have to add (blink-cursor-mode 0) explicitly to the end of my init file.

Thanks.

**Comment by jixiuf** [comment]
because emacs enable `blink-cursor-mode` by default .

**Comment by sutram** [comment]
@jixiuf Thanks for responding. I am using emacs 26.2 and before the merge, the cursor would not blink. I haven't upgraded emacs before or after the merge.

Thanks.

**Comment by akermu** [comment]
@sutram Maybe something changed in your configuration? You can disable cursor blinking by putting `(blink-cursor-mode -1)` in your file?

**Comment by Yevgnen** [comment]
Even having `(blink-cursor-mode -1)` in `init.el`, starting a `vterm` invalid the settings.

Since `blink-cursor-mode` is a global minor mode, I personally think packages or libraries should not change user settings.

**Comment by jubnzv** [comment]
I personally used the following solution for a quick fix:

```emacs-lisp
(defun jbz:vterm-restore-blink (buffer)
  "Disable blink-cursor-mode. Somehow vterm enables it on start."
  (blink-cursor-mode 0))

(add-to-list 'vterm-exit-functions #'jbz:vterm-restore-blink)
```

**Comment by sutram** [comment]
@jubnzv Thank you! I'll try your workaround. The blinking cursor is driving me crazy. Even adding (blink-cursor-mode 0) in the init file was not working after emacs session restore so I had to also add  blink-cursor-blinks 1 to the init file.

**Comment by jixiuf** [comment]
```

(define-minor-mode blink-cursor-mode
  "Toggle cursor blinking (Blink Cursor mode).

If the value of `blink-cursor-blinks' is positive (10 by default),
the cursor stops blinking after that number of blinks, if Emacs
gets no input during that time.

See also `blink-cursor-interval' and `blink-cursor-delay'.

This command is effective only on graphical frames.  On text-only
terminals, cursor blinking is controlled by the terminal."
  :init-value (not (or noninteractive
		       no-blinking-cursor
		       (eq system-type 'ms-dos)
		       (not (display-blink-cursor-p))))
  :initialize 'custom-initialize-delay
  :group 'cursor
  :global t
  (blink-cursor-suspend)
  (remove-hook 'after-delete-frame-functions #'blink-cursor--rescan-frames)
  (remove-function after-focus-change-function #'blink-cursor--rescan-frames)
  (when blink-cursor-mode
    (add-function :after after-focus-change-function #'blink-cursor--rescan-frames)
    (add-hook 'after-delete-frame-functions #'blink-cursor--rescan-frames)
    (blink-cursor--start-idle-timer)))
```
I try run `emacs -Q ` and then `(print blink-cursor-mode) `

the result is `t` on macOS ,so this is  enabled by emacs not by vterm,
vterm just respect  `blink-cursor-mode`
if you disable ` blink-cursor-mode` and call vterm again ,it won't blink

**Comment by dangom** [comment]
I can reproduce. After installing emacs-libvterm and calling vterm, the cursor starts to blink. It's definitely something related to the package.

**Comment by akermu** [comment]
Fixed with #104
